CHANGELOG.md

```diff
@@ -5,6 +5,9 @@
 
 * refactor from express to fastify (#2045)
 * refactor from Express to Fastify
+* API client library changes
+
+Note: the client version number is synchronized with the API's version number, necessitating a major version bump. This does _not_ require DB restore.
```

Comment:
We should use the [Note syntax specified here](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts), and set it just below the BREAKING CHANGES title